bin/wmagent-component-standalone

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+"""
+wmagent-component-standalone
+
+Utility script for running a WMAgent component standalone.
+"""
+
+import os
+import sys
+import logging
+import importlib
+
+from argparse import ArgumentParser
+from WMCore.Configuration import loadConfigurationFile
+from WMCore.ResourceControl.ResourceControl import ResourceControl
+from WMCore.Services.CRIC.CRIC import CRIC
+from WMCore.WMInit import connectToDB
+
+
+def createOptionParser():
+    """
+    _createOptionParser_
+
+    Creates an option parser to setup the component run parameters
+    """
+    optParser = ArgumentParser()
+    optParser.add_argument("--config", dest="wmaConfigPath", help="WMAgent configuration file.",
+                           default=os.environ.get("WMAGENT_CONFIG", None))
```

Comment:
Aren't we migrating to `WMA_CONFIG_FILE` env var?

Reply:
That's a good comment I was thinking about it already but did not push myself enough to switch to it at once. I'll change that.